conf/parse_test.go

```diff
@@ -383,3 +383,7 @@ func TestIncludeVariablesWithChecks(t *testing.T) {
 	expectKeyVal(t, m, "BOB_PASS", "$2a$11$dZM98SpGeI7dCFFGSpt.JObQcix8YHml4TBUZoge9R1uxnMIln5ly", 3, 1)
 	expectKeyVal(t, m, "CAROL_PASS", "foo", 6, 3)
 }
+
+func TestParser(t *testing.T) {
+	Parse(`A@@Føøøø?˛ø:{øøøø˙˙`)
```

Comment:
Maybe be a bit more explicit on what this test is doing (name the test appropriately) and check error returned, assuming one is expected :-)

Reply:
For instance, it could be `TestNoInfiniteLoop` or something like that. And for that test, check that returned error contains something about "Unexpected EOF"?

Reply:
Actually that one slipped in by accident. However, I'll keep it and fix as suggested